### PR TITLE
fix: return min-height for footer

### LINKED ISSUE #28: Career site fixes: Footer
**Problem**
On Career site, Positions list page:
- Footer have extra 16px spacing for the bottom part

CleanShot 31 10 2024 at 14-42

On Career site, Position page:
- Similar issue

**Solution**
Remove extra 16px spacing at the bottom of footer.

### PR DESCRIPTION
**Pull request by Lyams**
Closes #28.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/assets/stylesheets/src/career_site/_main.scss [hunk 1]
```diff
@@ -41,6 +41,10 @@ footer .container-fluid {
   min-height: calc(100% - #{$footer-min-height});
```
Comment (Lyams) [inline app/assets/stylesheets/src/career_site/_main.scss:41]: I forgot that we were using it here too when deleted it.

**Comment by ksnitsky** [comment]
If you only need this number to only place footer at the bottom, you can try to use `mt-auto` for example

**Comment by Lyams** [comment]
@PrinceYauj The code suggested by Konstantin works fine. But could you review also?

**Review by PrinceYauj — DISMISSED** [review]

**Comment by Lyams** [comment]
@PrinceYauj Resolve merge conflict. Please approve again.

**Pull request merged**